data/json/items/magazine/762.json

```
@@ -102,5 +102,57 @@
     "ammo_type": [ "762" ],
     "flags": [ "MAG_BULKY" ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "762": 75 } } ]
+  },
+  {
+    "id": "vz58mag10",
+    "looks_like": "stanag30",
+    "type": "MAGAZINE",
+    "name": { "str": "VZ 10-round magazine" },
+    "description": "A modified Czechoslovak rifle magazine which has been tweaked to only be capable of holding 10 rounds, in accordance with certain US magazine size restrictions.",
+    "variant_type": "gun",
```

```suggestion
    "description": "A modified Czechoslovak rifle magazine which has been tweaked to only be capable of holding ten rounds, in accordance with certain US magazine size restrictions.",
    "variant_type": "gun",
```
I don't think these magazines need a variant when there's nothing to confuse them with 😛